src/settings/spacesSettings.ts

```diff
@@ -0,0 +1,60 @@
+import { VenueTemplate } from "types/venues";
+
+import RoomIconArtPiece from "assets/icons/icon-room-artpiece.svg";
+import RoomIconAuditorium from "assets/icons/icon-room-auditorium.svg";
+import RoomIconBurnBarrel from "assets/icons/icon-room-burnbarrel.svg";
+import RoomIconConversation from "assets/icons/icon-room-conversation.svg";
+import RoomIconExperience from "assets/icons/icon-room-experience.svg";
+import RoomIconMap from "assets/icons/icon-room-map.svg";
+import RoomIconMusicBar from "assets/icons/icon-room-musicbar.svg";
+
+export interface VenueSpace {
```

Comment:
`VenueSpace` sounds like in between the old and the new. Suggesting something like `SpacesListItem` or `AvailableSpacesItem` or anything that adds more context to the Space concept.

Reply:
sounds better. fixed in 919b1f031c1e636c9de7dbd1aabb12ce7f337fef